code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -658,7 +658,7 @@ var/global/list/robot_modules = list(
 
 	return
 
-/obj/item/weapon/robot_module/security/combat
+/obj/item/weapon/robot_module/combat

```

Comment:
So _this_ is what was wrong with the combat module not spawning with any gear, huh?